src/main/java/com/znsio/teswiz/aspects/AspectJMethodLoggers.java

```diff
@@ -0,0 +1,38 @@
+package com.znsio.teswiz.aspects;
+
+import com.context.SessionContext;
+import com.context.TestExecutionContext;
+import org.apache.log4j.Logger;
+import org.aspectj.lang.JoinPoint;
+
+public class AspectJMethodLoggers {
+    private static final Logger LOGGER = Logger.getLogger(AspectJMethodLoggers.class.getName());
+    private final TestExecutionContext context;
+
+    public AspectJMethodLoggers() {
+        long threadId = Thread.currentThread().getId();
+        context = SessionContext.getTestExecutionContext(threadId);
+    }
+
+    public static void beforeAnyMethod(JoinPoint joinPoint) {
+        LOGGER.info(String.format("Entering method: %s ", joinPoint.getSignature().getName()));
```

Comment:
This approach still will result in 2 log messages when there are parameters. There should be only 1 log message per method

Reply:
Updated